app/authorities/sufia/authorities/local/find_works.rb

```diff
@@ -0,0 +1,21 @@
+module Sufia::Authorities
+  class Local::FindWorks < Qa::Authorities::Base
+    attr_reader :q
```

Comment:
@andjsmit Apparently you are doing it the recommended way.   This still seems like a bug in QA.  I see that this has been worked around in the  past like this: https://github.com/curationexperts/alexandria/blob/7e6dfafefb2cd9bc17aa956208e8a86495e5b1eb/app/authorities/local_names.rb#L16-L17